src/Admin/Views/Shared/_OrganizationForm.cshtml

```diff
@@ -83,14 +83,24 @@
                     @{
                         var planTypes = Enum.GetValues<PlanType>()
                             .Where(p =>
-                                Model.Provider == null ||
+                                (Model.Provider == null && !p.ToString().Contains("2020")) ||
                                 (Model.Provider != null
-                                 && p is >= PlanType.TeamsMonthly2019 and <= PlanType.EnterpriseAnnually2019 or >= PlanType.TeamsMonthly2020 and <= PlanType.EnterpriseAnnually)
+                                 && (
+                                     (Model.Provider.CreationDate < new DateTime(2023, 11, 6) && p.ToString().Contains("2020"))
```

Comment:
This is another good bit of evidence to support that we need some kind of central log/configuration with dates around pricing. Hardcoding the value all around is going to hurt us later when we inevitably have to adjust this all again.